### PR TITLE
Saturated tower codes update to L1TCaloLayer1 emulation

### PR DESCRIPTION
**Pull request by nsmith-**
This provides special codes for saturated tower, using spare values at the top of the 9-bit E+H value sent to Layer2.

### CODE REVIEW

**Comment by nsmith-** [comment]
Now configurable!
With the firmware version set to 0, we recover full bitwise agreement with `l1t-integration-v67.2` which is what this PR is based on. (no layer1 changes since then)

``` diff
diff --git a/L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py b/L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py
index 47295b7..2b71b02 100644
--- a/L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py
+++ b/L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py
@@ -18,5 +18,5 @@ simCaloStage2Layer1Digis = cms.EDProducer(
     unpackEcalMask = cms.bool(False),
     unpackHcalMask = cms.bool(False),
     # See UCTLayer1.hh for firmware version
-    firmwareVersion = cms.int32(1),
+    firmwareVersion = cms.int32(0),
     )
```

**Comment by nsmith-** [comment]
@rekovic  please merge
What CMSSW release can I expect this in?

**Comment by rekovic** [comment]
@nsmith-  Tag v75.0 was made after the attempt to merge this PR. 
Let me try again and make a new tag. Sorry about that.

**Comment by rekovic** [comment]
v75.0 did not contain the full merge of this PR

Now fully merged, and tagged as v77.0.

**Comment by rekovic** [comment]
@nsmith- Hope to make a PR of the l1t-integration into CMSSW by the end of this week.
